### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by boedy** [nginxinc/docker-nginx-amplify#2]
>  Launching amplify-agent ...
>
> Couldn't start the agent, please check /var/log/amplify-agent/agent.log

Work around:

```
# Install the Amplify Agent
RUN curl -L -o amplify-install.temp.sh \
    https://github.com/nginxinc/nginx-amplify-agent/raw/master/packages/install.sh && \
    head -n -12 amplify-install.temp.sh > amplify-install.sh && \
    rm amplify-install.temp.sh && \
    sh ./amplify-install.sh && \
    rm -f ./amplify-install.sh
```

**Comment by arun-gupta** [comment]
The instructions need to be updated to signup for an account at https://amplify.nginx.com/signup/, edit the `Dockerfile` and change API_KEY environment variable to specify your API key.

Then it built fine for me.

**Comment by ptreyes** [comment]
Thank you both! The entire build is gonna be updated shortly, and the docs will be corrected accordingly.

**Comment by cdarwin** [comment]
Closed by [7025f46d43](https://github.com/nginxinc/docker-nginx-amplify/compare/7025f46d4321aa60188d2bc22f991f8160f3de9f...master_
